stage2/01-sys-tweaks/01-run.sh

```diff
@@ -17,6 +17,8 @@ install -m 644 files/x11vnc.service	"${ROOTFS_DIR}/lib/systemd/system/"
 install -d				"${ROOTFS_DIR}/home/${FIRST_USER_NAME}/.vnc"
 install -m 644 files/passwd		"${ROOTFS_DIR}/home/${FIRST_USER_NAME}/.vnc/"
 
+cp -f "${ROOTFS_DIR}/usr/share/doc/avahi-daemon/examples/ssh.service" "${ROOTFS_DIR}/etc/avahi/services/"
```

Comment:
what if you use the `install` command like above?
that also makes sure the proper chmod is used;

Reply:
makes sense will switch